aws/resource_aws_spot_fleet_request_test.go

```diff
@@ -187,8 +337,10 @@ func TestAccAWSSpotFleetRequest_fleetType(t *testing.T) {
 			{
 				Config: testAccAWSSpotFleetRequestConfigFleetType(rName, rInt, validUntil),
 				Check: resource.ComposeAggregateTestCheckFunc(
-					testAccCheckAWSSpotFleetRequestExists(resourceName, &sfr),
-					resource.TestCheckResourceAttr(resourceName, "spot_request_state", "active"),
+					testAccCheckAWSSpotFleetRequestExists(
+						resourceName, &sfr),
+					resource.TestCheckResourceAttr(
+						resourceName, "spot_request_state", "active"),
```

Comment:
I'm guessing the original pull request had these split due to line length, can you please revert these here and below so they do not show as a difference? Thanks!